content/en/getting_started/integrations/aws.md

```diff
@@ -76,43 +76,38 @@ Before getting started, ensure you have the following prerequisites:
     * secretsmanager:PutSecretValue
     * serverless:CreateCloudFormationTemplate
 
-
 ## Setup
 
+2. Go to the the [AWS integration configuration page][8] in Datadog and click **Add AWS Account**.
 
-2. From the AWS tile on the [Integrations page][8] in your Datadog account, select the Datadog products you wish to integrate with this AWS Account. This selects the correct default settings for integrating data from this AWS account for those products. These settings can be changed in the future if needed.
-{{< img src="getting_started/integrations/cloudformation-setup.png" alt="The Datadog AWS integration tile showing the options for establishing the integration. The Role Delegation tab is highlighted.">}}
-
-3. Select the AWS Region where the CloudFormation stack will be launched. This also sets where to create the Datadog Lambda Forwarder for sending AWS logs to Datadog (if you selected Log Management).
-
-    **Note**: CloudWatch metrics are collected from ALL AWS regions you are using regardless of the region you select.
-
-4. Select or create the Datadog API Key used to send data from your AWS account to Datadog.
+3. Configure the integration's settings under the **Automatically using CloudFormation** option.  
+    **a.** Select the AWS regions to integrate with.  
```

Comment:
My opinion: Don't bold the a., b., c., d. The indenting is enough to set it apart visually.

Reply:
Thanks for confirming! I wasn't sure about that myself. Updated.